run/helloworld/e2e_test_setup.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- # Build the renderer image
+  name: gcr.io/cloud-builders/docker:latest
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/helloworld-${_SUFFIX}', '.']
```

Comment:
Why do we use `GOOGLE_CLOUD_PROJECT` in some cases and `PROJECT_ID` in others? They represent the same thing right?

Reply:
Cloud build uses PROJECT_ID as the substitute variable; however DPEs use GOOGLE_CLOUD_PROJECT as the kokoro env var and recommended env var.